src/pulse.js

```diff
@@ -106,7 +106,7 @@ define(function (require) {
       this.dcGain.gain.value = 1.7 * (0.5 - this.w);
     } else {
       w.connect(this.dNode.delayTime);
-      var sig = new p5.SignalAdd(-0.5);
+      var sig = new Signal(-0.5);
```

Comment:
Good question. I think it could be either, and either way, it won't have the the `mult` method mentioned below so we could use [Multiply](https://tonejs.github.io/docs/13.8.25/Multiply) to accomplish that